os/board/rtl8720e/src/component/wifi/wpa_supplicant/wpa_supplicant/wifi_eap_config.c

```diff
@@ -298,11 +298,11 @@ int connect_by_open_system(char *target_ssid)
 				//printf("\r\n[EAP]Associate with AP success\n");
 				break;
 			}
+			retry_count --;
 			if (retry_count == 0) {
 				//printf("\r\n[EAP]Associate with AP failed %d\n", ret);
```

Comment:
In these failure scenario (Associate with AP failed) , I think it is better to print the message.

Reply:
It's a BSP code, we only want to fix a SVACE issue in this PR. Let's give your opinion to Realtek.
@gexuyan How do you think Thapa's opinion?

Reply:
Hi @sunghan-chang ,
Sure, can only fix SVACE issues in this PR, I will add the logs in future PR.